docs/source/channels.rst

```diff
@@ -0,0 +1,99 @@
+Channels: Futures Shared Between Clients
+===================================
+
+Often it is desirable to respond to events from outside the distributed cluster 
+or to instantiate a new client in order to check on the progress of a set of tasks. 
```

Comment:
Would prefer to start with a one-line description of what channels do, then going into examples, then go into a greater description.  Your one line description below "A changing stream of futures shared between clients" looks great to me.  Can we elevate it to the first line?